vtex/billing-app
# Add vtex invoice provider to free apps

#### What is the purpose of this pull request?
Update the zero-value policy

#### What problem is this solving?
Purchase of free apps are made with zero-value policies, which must be updated

#### How should this be manually tested?
1. Clone/pull this repo/branch and link to your workspace
2. Try to install a free app
```shell
vtex install vtex.payment-provider-test-suite -r vtex --verbose
```
3. The installation should be successfull

#### Screenshots or example usage

#### Types of changes
